src/internal/operators/throwIfEmpty.ts

```diff
@@ -16,32 +18,57 @@ import { MonoTypeOperatorFunction } from '../types';
  *
  * const click$ = fromEvent(button, 'click');
  *
- * clicks$.pipe(
+ * click$.pipe(
  *   takeUntil(timer(1000)),
  *   throwIfEmpty(
  *     () => new Error('the button was not clicked within 1 second')
  *   ),
  * )
  * .subscribe({
  *   next() { console.log('The button was clicked'); },
- *   error(err) { console.error(err); },
+ *   error(err) { console.error(err); }
  * });
  * ```
  *
  * @param {Function} [errorFactory] A factory function called to produce the
```

Comment:
```suggestion
 * @param errorFactory A factory function called to produce the
```

We don't actually need this additional typing information now, it's just from legacy docs. (not your doing, just adding that change in as I notice it)